client/my-sites/stats/pages/providers/global-provider.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useState, ReactNode, useEffect } from 'react';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { default as usePlanUsageQuery } from './../../hooks/use-plan-usage-query';
+
+type StatsGlobalValuesProviderProps = {
+	children: ReactNode;
+};
+
+// Create a context for the provider
+export const StatsGlobalValuesContext = createContext< boolean | undefined >( undefined );
```

Comment:
Thanks for the suggestion. I think we can use an object when we need another value, and it will be a simple change. It's a good point that I could initialise it with `false`.